sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_manage_custom_models_async.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_manage_custom_models_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to manage the custom models on your account.
+USAGE:
+    python sample_manage_custom_models_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+"""
+
+import os
+import asyncio
+
+
+class ManageCustomModelsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    async def manage_custom_models(self):
+        # TODO: this can be used as examples in sphinx
+        from azure.core.credentials import AzureKeyCredential
+        from azure.core.exceptions import ResourceNotFoundError
+        from azure.ai.formrecognizer.aio import FormTrainingClient
+
+        async with FormTrainingClient(endpoint=self.endpoint, credential=AzureKeyCredential(self.key)) as form_training_client:
+            # First, we see how many custom models we have, and what our limit is
+            account_properties = await form_training_client.get_account_properties()
+            print("Our account has {} custom models, and we can have at most {} custom models".format(
+                account_properties.custom_model_count, account_properties.custom_model_limit
+            ))
+
+            # Next, we get a paged list of all of our custom models
+            custom_models = form_training_client.list_model_infos()
+
+            print("We have models with the following ids:")
+
+            # Let's pull out the first model
+            first_model = None
+            async for model in custom_models:
+                print(model.model_id)
```

Comment:
if you just want to pull out the first model, you can do this:

```
model = next(custom_models)
print(model.model_id)
```

Reply:
That's what I have in the sync code, but I had tried that in the async code and got TypeError: 'AsyncItemPaged' object is not an iterator. I might try harder and see if there's a nicer way though